medusa/status.py

```diff
@@ -23,15 +24,22 @@
 TIMESTAMP_FORMAT = '%Y-%m-%d %H:%M:%S'
 
 
-def status(config, backup_name):
+def status(config, backup_name, output):
     with Storage(config=config.storage) as storage:
 
         try:
             cluster_backup = storage.get_cluster_backup(backup_name)
         except KeyError:
             logging.error('No such backup')
+            if output == 'json':
+                print(json.dumps({}))
```

Comment:
I'm also finding it odd that we print the `{}` on error. I suppose this is what you meant and need, but still I'd expect it to be rather quiet than an empty dict.